nodes/widgets/locales/en-US/ui_button.html

```diff
@@ -18,6 +18,12 @@ <h3>Properties</h3>
     <h3>Dynamic Properties (Inputs)</h3>
     <p>Any of the following can be appended to a <code>msg.</code> in order to override or set properties on this node at runtime.</p>
     <dl class="message-properties">
+        <dt class="optional">label <span class="property-type">string</span></dt>
+        <dd>Override the label displayed on the button</dd>
+        <dt class="optional">icon <span class="property-type">string</span></dt>
+        <dd>Override the icon defined in the initial configuration</dd>
+        <dt class="optional">iconPosition <span class="property-type">'left' | 'right'</span></dt>
+        <dd>Change which side of the label the icon renders</dd>
```

Comment:
No mention of `msg.ui_control` - users will likely stumble here Joe.

In https://github.com/FlowFuse/node-red-dashboard/pull/861 you have added a comment above the dynamic properties to clarify this - see here: https://github.com/FlowFuse/node-red-dashboard/pull/861#discussion_r1618566423